homeassistant/components/brother/utils.py

```diff
@@ -5,22 +5,23 @@
 from pysnmp.hlapi.asyncio.cmdgen import lcd
 
 from homeassistant.const import EVENT_HOMEASSISTANT_STOP
-from homeassistant.core import callback
+from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers import singleton
+from homeassistant.helpers.typing import EventType
 
 from .const import DOMAIN, SNMP
 
 _LOGGER = logging.getLogger(__name__)
 
 
 @singleton.singleton("snmp_engine")
-def get_snmp_engine(hass):
+def get_snmp_engine(hass: HomeAssistant) -> hlapi.SnmpEngine:
     """Get SNMP engine."""
     _LOGGER.debug("Creating SNMP engine")
     snmp_engine = hlapi.SnmpEngine()
 
     @callback
-    def shutdown_listener(ev):
+    def shutdown_listener(ev: EventType) -> None:
```

Comment:
We prefer to use the core classes directly over the helpers that are just copying the core classes directly. We're migrating away from those helpers.
```suggestion
    def shutdown_listener(ev: Event) -> None:
```